lib/logproto/tests/test-regexp-multiline-server.c

```diff
@@ -58,14 +63,22 @@ test_lines_separated_with_prefix(LogTransportMockConstructor log_transport_mock_
   assert_proto_server_fetch(proto, "Foo Multiline\nmulti", -1);
 
   log_proto_server_free(proto);
-  log_proto_server_options_destroy(&proto_server_options);
   multi_line_regexp_free(re);
 }
 
-Test(log_proto, test_lines_separated_with_prefix)
-{
-  test_lines_separated_with_prefix(log_transport_mock_stream_new);
-  test_lines_separated_with_prefix(log_transport_mock_records_new);
+ParameterizedTestParameters(log_proto, test_lines_separated_with_prefix) {
+	static LogTransportMockConstructorParam log_transport_mock_new_data_list[] = {
+		log_transport_mock_stream_new,
+		log_transport_mock_records_new,
+	};
+
+	return cr_make_param_array(
+			LogTransportMockConstructorParam,
+			log_transport_mock_new_data_list, sizeof (log_transport_mock_new_data_list) / sizeof (log_transport_mock_new_data_list[0]));
```

Comment:
You can use `G_N_ELEMENTS(log_transport_mock_new_data_list)` to calculate the size.

https://developer.gnome.org/glib/stable/glib-Standard-Macros.html#G-N-ELEMENTS:CAPS